english/Pam_and_Jim

```diff
@@ -0,0 +1 @@
+Pam and Jim are the greatest couple in the history of tv!
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.